tests/tests.js

```diff
@@ -869,6 +869,32 @@ describe('namedGroup', () => {
 		});
 		assert.strictEqual(expected, transpiled);
 	})
+
+	it('should not throw when namedGroup is not enabled', () => {
+		let transpiled;
+		assert.doesNotThrow(() => {
+			transpiled = rewritePattern('(?<name>.)\\k<name>', '');
+		});
```

Comment:
Can you also add an assertion which checks that `transpiled` is correct? (i.e. that it has not been transformed)

Reply:
ok